src/javascripts/components/options-table.js

```diff
@@ -0,0 +1,34 @@
+var OptionsTable = {
+  init: function () {
+    OptionsTable.expandMacroOptions()
+  },
+  // Open Nunjucks tab and expand macro options details when URL hash is 'nunjucks-options-[example]'
+  expandMacroOptions: function () {
+    var hash = window.location.hash
+
+    if (hash.match('^#options-')) {
+      var exampleName = hash.split('#options-')[1]
+
+      if (exampleName) {
+        var tabLink = document.querySelector('a[href="#' + exampleName + '-nunjucks"]')
+        var tabHeading = tabLink ? tabLink.parentNode : null
+        var optionsDetailsElement = document.getElementById(hash.substring(1) + '-details')
+
+        if (tabHeading && optionsDetailsElement) {
+          var tabsElement = optionsDetailsElement.parentNode
+
+          tabLink.setAttribute('aria-expanded', true)
```

Comment:
This seems a little fragile as it'd be easy to update the tabs component and forget to update this, which could break this. I think long term we should make it possible to tell the tabs component to open a particular tab.

I think it's _probably_ OK to go with this for now though?

Reply:
as suggested above we could add a click on the element and then tabs.js will handle the rest?